1-mandatory/3-slideshow/slideshow.js

```diff
@@ -1 +1,88 @@
 // Write your code here
+let imgArr = [
+   {
+     src: "https://images.unsplash.com/photo-1546069901-ba9599a7e63c?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxzZWFyY2h8Mnx8Zm9vZHxlbnwwfHwwfHw%3D&auto=format&fit=crop&w=500&q=60",
+     alt: "salad dish",
+   },
+   {
+     src: "https://images.unsplash.com/photo-1565958011703-44f9829ba187?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MTB8fGZvb2R8ZW58MHx8MHx8&auto=format&fit=crop&w=500&q=60",
+     alt: "cake image",
+   },
+
+   {
+     src: "https://images.unsplash.com/photo-1467003909585-2f8a72700288?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MTZ8fGZvb2R8ZW58MHx8MHx8&auto=format&fit=crop&w=500&q=60",
+     alt: "sea food image",
+   },
+
+   {
+     src: "https://images.unsplash.com/photo-1473093295043-cdd812d0e601?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MjV8fGZvb2R8ZW58MHx8MHx8&auto=format&fit=crop&w=500&q=60",
+     alt: "pasta image",
+   },
+   {
+     src: "https://images.unsplash.com/photo-1565299507177-b0ac66763828?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxzZWFyY2h8NDR8fGZvb2R8ZW58MHx8MHx8&auto=format&fit=crop&w=500&q=60",
+     alt: "berger",
+   },
+ ];
+ let divEl = document.getElementById("slideshow");
+ let imgEl = document.createElement("img");
+ divEl.appendChild(imgEl);
+ imgEl.className = "slideImg";
+
+ let backBtn = document.getElementById("back");
+ let forwardBtn = document.getElementById("forward");
+ let stopBtn = document.getElementById("stop");
+ let autoBackBtn = document.getElementById("autoBack");
+ let autoForwardBtn = document.getElementById("autoForward");
+ let input = document.getElementById("input");
+ let delaySetBtn = document.getElementById("delaySet");
+ let delay = 1000;
+
+ forwardBtn.addEventListener("click", () => {
+   goForward();
+ });
+
+ backBtn.addEventListener("click", () => {
+   goBack();
+ });
+
+ autoForwardBtn.addEventListener("click", () => {
+   autoForwardInt = setInterval(() => {
+     goForward();
+   }, delay);
+ });
+
+ autoBackBtn.addEventListener("click", () => {
+   autoBackInt = setInterval(() => {
+     goBack();
+   }, delay);
+ });
```

Comment:
Once you set an interval, the interval exists outside the scope of the variable you assign it to (it exists running inside the browser), and the variable is actually just a reference to the interval. This means that every time you call `setInterval`, even if you're overwriting the reference to it in code, you create _another_ interval. You can try this out in your browser by opening developer console and pasting:
```
let x = setInterval(() => console.log('hi'), 5000)
x = setInterval(() => console.log('hi'), 5000)
```
You'll notice that `hi` is logged twice every 5s instead of once.

To fix this, be sure to `clearInterval(autoForwardInt)` first before assigning it to be a reference to a new interval.